City-of-Helsinki/drupal-module-helfi-tpr
# Uhf 3450 set school year

Adds school year selection, uses it at school search filters and moves the filters to separate submodule.

**How to test:**

1. Setup up Kasko development environment.
2. Check out Kasko branch `UHF-3450-kasko-set-school-year`.
3. Get the changes for this module: `composer require drupal/helfi_tpr:dev-UHF-3450-set-school-year`.
4. Get changes to translations: `composer require drupal/helfi_platform_config:dev-UHF-3450-school-year-translations`.
5. Import configs: `drush cim`.
6. Clear caches: `drush cr`.
7. Import admin page translations: `drush locale-import fi /app/public/modules/contrib/helfi_platform_config/translations/fi/strings.po`.
8. Make sure you have high school search page ready with published high schools. Also make sure you have imported `tpr_ontology_word_details`. If not, check [this previous PR](https://github.com/City-of-Helsinki/drupal-module-helfi-tpr/pull/83) for instructions.
9. Go to the high school search. There shouldn't be any filter values (except "All") since the school year is not yet set.
10. Now as an admin or content editor, you should be able to visit `/admin/config/school-settings`. Set the school year to `2021-2022`.
11. Now revisit the high school search and there should be filter values present.
